gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -951,6 +954,11 @@ export class AdvancedDynamicTexture extends DynamicTexture {
      */
     public parseContent(serializedObject: any) {
         this._rootContainer = Control.Parse(serializedObject.root, this) as Container;
+        const width = serializedObject.width;
+        const height = serializedObject.height;
+        if (width && height) {
```

Comment:
this will only rescale if both are not 0. is that the expected behavior?

Reply:
Should i change it !== null instead. Want to cover the case for previous jsons where this is not defined.

Reply:
you know how you want to deal with it :-)

You can check the typeof width and typeof height (should be be 'number') instead. And that they are >= 0